ranker/shared/omnicorp_postgres.py

```diff
@@ -58,6 +58,30 @@ async def get_prefixes(self):
         prefixes = [ row['table_name'] for row in rows ]
         return prefixes
 
+    async def get_pmids(self, node):
+        """Get shared PMIDs."""
+        prefix = get_postgres_curie_prefix(node)
+        if (
+                prefix not in self.prefixes
+        ):
+            return 0
```

Comment:
Is returning 0 the right thing?  It looks like this function can either return a list of curies, an empty list, None, or 0.  I think None and 0 should be replaced with [].